packages/core/contracts-ovm/insured-bridge/implementation/OVM_BridgeDepositBox.sol

```diff
@@ -221,7 +228,7 @@ contract OVM_BridgeDepositBox is OVM_CrossDomainEnabled, OVM_Testable {
 
         StandardBridgeLike(Lib_PredeployAddresses.L2_STANDARD_BRIDGE).withdrawTo(
             l2Token, // _l2Token. Address of the L2 token to bridge over.
-            bridgeRouter, // _to. Withdraw, over the bridge, to the l1Withdraw contract.
+            whitelistedTokens[l2Token].l1BridgePool, // _to. Withdraw, over the bridge, to the l1 withdraw contract.
```

Comment:
@chrismaree does this slight change affect any e2e tests? i don't think so but want extra eyes on it

Reply:
there are no e2e tests for this part just yet :) as long as we've got the unit test for this updated (which you did) we good to go